packages/@lwc/integration-karma/test/shadow-dom/Event-properties/Event.currentTarget.spec.js

```diff
@@ -3,44 +3,55 @@ import { createElement } from 'lwc';
 import Container from 'x/container';
 
 describe('Event.currentTarget', () => {
-    it('should be null when accessed asynchronously', function (done) {
+    it('should be null when accessed asynchronously', async () => {
         const container = createElement('x-container', { is: Container });
         document.body.appendChild(container);
 
-        container.addEventListener('test', (event) => {
-            expect(event.currentTarget).toEqual(container);
-            setTimeout(() => {
-                expect(event.currentTarget).toBeNull();
-                done();
+        const [first, second] = await new Promise((resolve) => {
+            container.addEventListener('test', (event) => {
+                const first = event.currentTarget;
+                setTimeout(() => {
+                    const second = event.currentTarget;
+                    resolve([first, second]);
+                });
             });
+            const div = container.shadowRoot.querySelector('div');
+            div.dispatchEvent(new CustomEvent('test', { bubbles: true, composed: true }));
         });
-        const div = container.shadowRoot.querySelector('div');
-        div.dispatchEvent(new CustomEvent('test', { bubbles: true, composed: true }));
+
+        expect(first).toEqual(container);
```

Comment:
All of these should be `toBe`. The tests pass either way.

This is not your issue, but we may as well clean it up while we're here. 🙂 

```suggestion
        expect(first).toBe(container);
```